zsh-wakatime.plugin.zsh

```diff
@@ -19,4 +23,8 @@ waka_filename() {
     fi
 }
 
-precmd_functions+=(send_wakatime_heartbeat)
+heartbeat() {
+  (send_wakatime_heartbeat > /dev/null 2>&1 &)
```

Comment:
Just move `&` below and call `send_wakatime_heartbeat` directly so this function isn't needed anyore.